bin/alluxio-masters.sh

```diff
@@ -10,6 +10,8 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
+set -o pipefail
```

Comment:
the main reason is that we use pipeline in the shell command, if any command in the pipeline fails, the correct exit code "non-zero" will be returned if we set "set -o pipefail",  otherwise,  it may return 0 as a wrong exit code. Fox example: 
command 1:
omand_exit_with_nonzero | echo 1
echo $?

It shows a wrong exit code '0'

command 2:
set +o pipefail
comand_exit_with_nonzero | echo 1

it  shows a correct exit code '127'

Reply:
does setting `pipefail` here improve the behavior of `alluxio-masters.sh`? If not I'd suggest leaving it out since we don't use it for other scripts

Reply:
Sorry, some mistakes in the 2 examples, you can try them again to see the difference!

Reply:
Also，we can run "man set" on linux to find the usage below:
![MacHi 2019-03-20 14-09-09](https://user-images.githubusercontent.com/10387174/54662894-81c64b80-4b1a-11e9-9556-0befcfec1035.png)